app/src/main/java/com/example/tasky/feature_agenda/data/local/relation/EventAttendeeCrossRef.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.tasky.feature_agenda.data.local.relation
+
+import androidx.room.Entity
+
+@Entity(primaryKeys = ["eventId","attendeeId"])
+data class EventAttendeeCrossRef(
+    val eventId: Int,
+    val attendeeId: Int
+)
```

Comment:
I'd make the event-attendee relation is a one-to-many relation which keeps things a lot simpler. You could argue that one attendee can join multiple events, but I think in this case this rather adds unnecessary overhead, since attendees are quite simple data structures. Having a many to many relation would add a lot of these cross refs and also make handling the DB more complex without real advantages